efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -161,19 +88,75 @@ def build(self, feats_shape):
                                            self.data_format,
                                            self.strategy,
                                            name='op_after_combine{}'.format(
-                                               len(feats_shape)))
-    self.built = True
+                                               len(input_shape)))
+
+    if self.weight_method == 'attn':
```

Comment:
I won't approve such modifies that just wrapper it with a keras layer and change code positions.